models/staging/ecommerce/stg_ecommerce__product_category_name_translation.sql

```diff
@@ -0,0 +1,13 @@
+
+select
+    -- primary key
+
+    -- foreign keys
+
+    -- timestamps
+
+    -- dimensions
+    string_field_0,
+    string_field_1
```

Comment:
it might be useful to alias this for use later on

Reply:
Alias added as product_category_name_portuguese, product_category_name_english with commit [e51990f](https://github.com/mtla-saurabh/ae-capstone-sshahdeo/commit/e51990f08f496ab9c2bfbf3e35ddf29c72b95d6e)